warehouse/accounts/security_policy.py

```diff
@@ -16,13 +16,14 @@
     SessionAuthenticationHelper,
     extract_http_basic_credentials,
 )
+from pyramid.authorization import ACLHelper
 from pyramid.httpexceptions import HTTPUnauthorized
-from pyramid.interfaces import IAuthorizationPolicy, ISecurityPolicy
-from pyramid.threadlocal import get_current_request
```

Comment:
I bet this made you so happy @dstufft

Reply:
![](https://media.giphy.com/media/gVoBC0SuaHStq/giphy.gif)